vNext/src/ClientGenerator/BootstrapCodegen.proj

```diff
@@ -1,39 +1,79 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
```

Comment:
There is the key you're saying: "same solution"...this does not have to do anything with that.